### PR TITLE
Custom subtitles styles

### PR DESCRIPTION
**Pull request by geriux**
Support for custom styles for the text tracks.

### CODE REVIEW

File: ios/Video/RCTVideo.m [hunk 1]
```diff
@@ -1082,6 +1084,16 @@ - (void) setSideloadedText {
     }
     [_player.currentItem.tracks[i] setEnabled:isEnabled];
   }
+
+  if ([_textTrackStyles count]> 0) {
```
Comment (fluiddot) [inline ios/Video/RCTVideo.m]: I'd add a space => `[_textTrackStyles count] > 0`

File: ios/Video/RCTVideo.m [hunk 2]
```diff
@@ -1082,6 +1084,16 @@ - (void) setSideloadedText {
...
+    AVTextStyleRule *textStyle = [[AVTextStyleRule alloc]initWithTextMarkupAttributes:@{
+                                (id)kCMTextMarkupAttribute_ForegroundColorARGB : @[ @1, @1, @1, @1 ],
+                                (id)kCMTextMarkupAttribute_BackgroundColorARGB : @[ @0, @0, @0, @0],
```
Comment (fluiddot) [inline ios/Video/RCTVideo.m]: In the android implementation you set alpha `160` in a color (not sure if it was foreground or background) but here I don't see any opacity change, is that ok?

Comment (geriux) [reply ios/Video/RCTVideo.m]: Yup it is ok, Android and iOS calculate the background differently for the subtitles 🤯

File: ios/Video/RCTVideo.m (hunk 1 above)
Comment (geriux) [reply ios/Video/RCTVideo.m]: Updated! thanks!!

**Comment by fluiddot** [comment]
Ready to merge!

**Pull request merged**